### ISSUE TITLE
Great Project

### ISSUE THREAD

**Issue by tobiz** [jonmatifa/zfsmanager#8]
Would be interested to see this get to 'production' status. A simple ZFS setup is not too difficult but anything that makes it easier to control all the options has to be a great idea, especially via webmin.

**Comment by UKROB** [comment]
I agree, I'm absolutely excited to see this reach production, this for me would be the most valuable addon for webmin and if it's anything like the FreeNAS approach I'd be over the moon!

**Comment by datenheim** [comment]
please don't use an issue tracker for saying thank you - because that is not an issue ;)
